SetCurrentVersion.cmd

```diff
@@ -1,4 +1,4 @@
 set MAJOR=1
-set MINOR=4
-set PATCH=34
+set MINOR=5
+set PATCH=0
```

Comment:
The minor version was intended to stay in sync with the SARIF version. But we moved off 1.4. My care factor here is low.